extensions/gadgets/AdviceBar/AdviceBar.html

```diff
@@ -1,3 +1,5 @@
+<script src="{{cache_slug}}/extensions/gadgets/AdviceBar/AdviceBar.js"></script>
```

Comment:
Yeah, this was similar to my other question on why not use ``cache_slug`` instead of ``ASSET_DIR_PREFIX``--I see it's based on the directory again. Thanks for the clarification.